app/src/main/res/values/themes.xml

```diff
@@ -28,14 +28,15 @@
         <item name="android:fontFamily">@font/inter</item>
     </style>
 
-    <style name="home_20_sp">
+    <style name="home_30_sp">
```

Comment:
style name is home_30_sp but dimension is <item name="android:textSize">@dimen/dimens20sp</item>